### DISCUSSION TITLE
interrupt by timer

### DISCUSSION THREAD

**Discussion by yuranikspb** [SpenceKonde/megaTinyCore#402] [Q&A]
i want to useit to read pins in interrupt
any examples?

**Comment by HolleL** [comment]
Hello yuranikspd
There is an example for periodic interrupt with the RTC-Timer in [TB3213 - Getting Started with RTC](http://ww1.microchip.com/downloads/en/Appnotes/TB3213-Getting-Started-with-RTC-DS90003213.pdf)
Is this what you search ? 

The example for Arduino [RTC_Periodic_Interrupt.ino.txt](https://github.com/SpenceKonde/megaTinyCore/files/6484712/RTC_Periodic_Interrupt.ino.txt)

**Reply by yuranikspb** [reply]
"Is this what you search ?"
Yes ! thnx!
